0.25.3 Release Notes (#401)
astronomer/docs: cloud/stable/07_resources/01_release-notes.md

## Astronomer v0.25

### v0.25.3

> Release Date: June 28th, 2021

Reviewer: ```suggestion
Release Date: June 30th, 2021
```

Change this, or just wait until release-day to finalize?

> ### Bypass User Invite Emails via Houston API

Reviewer: ```suggestion
### Bypass Email Verification for User Invites via Houston API
```

> The Houston API `workspaceAddUser` mutation now includes a `bypassInvite` field. When this field is set to true, users invited to a Workspace no longer need to first verify their email addresses before accessing the Workspace. This type of query can be useful to minimize friction when programmatically inviting many users to your platform. For more information, see [Sample Mutations](/docs/enterprise/v0.25/manage-astronomer/houston-api#sample-mutations).

Reviewer: ```suggestion
The Houston API `workspaceAddUser` mutation now includes a `bypassInvite` field. When this field is set to true, users invited to a Workspace no longer need to first verify their email addresses before accessing the Workspace. This type of query can be useful to minimize friction when programmatically inviting many users to Astronomer. For more information, see [Sample Mutations](/docs/cloud/stable/manage-astronomer/houston-api#sample-mutations).
```

If we're keeping this for Cloud, I think change to `Astronomer` over `your Platform`

### Minor Improvements

> - Workspace Admins can now perform CRUD operations on any Deployment within their Workspace, even if they don't have Deployment Admin permissions for the given Deployment.

Reviewer: ```suggestion
- Workspace Admins can now see and perform CRUD operations on any Deployment within their Workspace, even if they don't have Deployment Admin permissions for the given Deployment.
```

Author: I think the ability to do CRUD operations assumes seeing Deployments

Reviewer 2: Yes. You are right @jwitz

### Bugfixes

- Fixed broken links to Deployments in alert emails.
- Fixed an issue where historical logs did not appear in the Astronomer UI.
- Fixed an issue where long task execution logs did not appear in the Airflow UI.

> - Fixed a visual bug where some Deployments with only 1 Scheduler were shown has having 2 in the Astronomer UI.

Reviewer: ```suggestion
- Fixed a visual bug where some Deployments with only 1 Scheduler were shown as having 2 in the Astronomer UI.
```

> - Fixed a visual bug where users without Workspace Admin permissions had a non-functional Invite Users button in the Astronomer UI.

Reviewer: ```suggestion
- Fixed a visual bug where users without Workspace Admin permissions had a non-functional **Invite Users** button in the Astronomer UI.
```

## Astronomer v0.25

### v0.25.3

> Release Date: June 28th, 2021

Reviewer 2: Release is going to be on June 30th

Reviewer: Looks like there's a double entry for `## 0.25.3` - should only be one.

Reviewer 2: Cloud release is also going to be on June 30th

## v0.25.3

> Release Date: June 25, 2021

Reviewer 2: Release date - June 30th.

### Bypass User Invite Emails via Houston API

> The Houston API `workspaceAddUser` mutation now includes a `bypassInvite` field. When this field is set to true, users invited to a Workspace no longer need to first verify their email addresses before accessing the Workspace. This type of query can be useful to minimize friction when programmatically inviting many users to your platform. For more information, see [Sample Mutations](/docs/enterprise/v0.25/manage-astronomer/houston-api#sample-mutations).

Reviewer: I'm not actually sure that this is applicable to Cloud customers since it's a platform-wide setting (that we won't be enabling in Cloud). Is that right, @vishwas-astro ?

Reviewer 2: I am trying to remember how it was implemented. @aliotta is this API exposed for workspace admins regardless of whether the environment is cloud or enterprise?

Reviewer 3: yeah this will work for any workspace admin regardless of cloud or enterprise.

### Minor Improvements

- Workspace Admins can now perform CRUD operations on any Deployment within their Workspace, even if they don't have Deployment Admin permissions for the given Deployment.

> - Changed the default Scheduler count for new Deployments to 2.

Reviewer 2: I think this was always the case. The bug was when a 1.10.x deployment was upgraded to 2.x and there will be only 1 scheduler running but the UI will show 2. Please work with @bote795 to confirm and update the note here.

Reviewer 4: @vishwas-astro you are correct, for `Changed the default Scheduler count for new Deployments to 2.` it The bug was when a 1.10.x deployment was upgraded to 2.x and there will be only 1 scheduler running but the UI will show 2

### Bugfixes

- Fixed broken links to Deployments in alert emails.
- Fixed an issue where historical logs did not appear in the Astronomer UI.
- Fixed an issue where long task execution logs did not appear in the Airflow UI.
- Fixed a visual bug where users without Workspace Admin permissions had a non-functional Invite Users button in the Astronomer UI.

> ### v0.25.1

Reviewer: I see 0.25.2 in Enterprise release notes, but latest is 0.25.1 here - we should look back at release notes and add them here if relevant, or just a note that it was an "Enterprise-only" release.

https://github.com/astronomer/astronomer/releases

Reviewer 2: 0.25.2 was an enterprise only release

Reviewer: Maybe just add a quick note here, @jwitz ?
